### ISSUE TITLE
No download Option In Manga World

### ISSUE THREAD

**Issue by raffyOmega** [jakepurple13/OtakuWorld#114]
hello Admin, can u put some download option in mangaworld? for offline reading , We dont have a internetion connection in our home, Im using data connection only...

**Comment by jakepurple13** [comment]
Yeah sure. Currently, you can download each page if you press and hold on it, but I do have something in the works for a full chapter download.

**Comment by jakepurple13** [comment]
Feature added in 18.7.
To download chapter, when looking at the details, swipe a chapter left and download.
I do plan on changing how the chapter items look that make the extra options easier to see.
